### ISSUE TITLE
Either: git folder needs to be renamed to "Concorde" or all the model files need their paths changed as the default name is "Concorde-flightgear"

### ISSUE THREAD

**Issue by rhysackerman** [fgfs-fmanu/Concorde-Flightgear#1]
This causes the blue/yellow glider to be displayed instead of the actual model.

**Comment by fgfs-fmanu** [comment]
Does it works if you change the plane folder to "Concorde"? 
Your path sould be like that: ../''your aicrafts folder''/Concorde/''all the files and folders of the plane''

Please if other users have the same issue, please tell me. 

You can tell the problem on the forum if you wants.

I follow the problem. Have a good fixing. :)

**Comment by rhysackerman** [comment]
Yes, when renaming the folder to just "Concorde" it works fine. Might be an
idea to include this in the ReadMe as other users may not realise.

**Comment by fgfs-fmanu** [comment]
Ok. I'll do it. :)
